esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -0,0 +1,884 @@
+    last_access: 2019-11-29
+    info: Use automatic download.
+
+  MLS-AURA:
+    tier: 3
+    source: https://disc.gsfc.nasa.gov/datasets/ML2RHI_004/summary https://disc.gsfc.nasa.gov/datasets/ML2T_004/summary
+    last_access: 2020-02-03
+    info: |
+      Select "Data Access" -> "Subset/Get Data" -> "Get Data" and follow the
+      "Instructions for downloading". All *.he5 files need to be saved in the
+      $RAWOBS/Tier3/MLS-AURA directory, where $RAWOBS refers to the RAWOBS
+      directory defined in the user configuration file. Apply this procedure to
+      both links provided above. The temperature fields are necessary for quality
+      control of the RHI data (see Data Quality Document for MLS-AURA for more
+      information).
+      A registration is required
+
+  MODIS:
+    tier: 3
+    source: https://ladsweb.modaps.eosdis.nasa.gov/search/order
+    last_access: 2019-02-09
+    info: |
+      In Products: select "MODIS Aqua", "Collection 6.1" and
+        "L3 Atmosphere Product", click on MYD08_M3.
+      In Time: select from 2000-01-01 to today.
+      In Location: skip, the global domain will be applied.
+      In Files: select all.
+      Submit the order.
+      A registration is required to download the data.
+
+      Caveats
+        clwvi and clivi data are in-cloud values whereas CMIP5 models provide
+        grid-box averages --> multiply MODIS clwvi and clivi values with cloud
+        fraction as a first guess
+
+  MTE:
+    tier: 3
+    source: http://www.bgc-jena.mpg.de/geodb/BGI/Home
+    last_access: 2019-05-07
+    info: |
+      Download the following files:
+        EnsembleGPP_GL.nc
+      A registration is required for
+
+  NCEP:
+    tier: 2
+    source: http://www.esrl.noaa.gov/psd/data/gridded/data.ncep.reanalysis.html
+    last_access: 2019-02-04
+    info: |
+      To facilitate the download, the links to the ftp server are provided.
+      Since the filenames are sometimes identical across different
+      save the data in two subdirectories in input_dir_path.
+      Subdirectory pressure/:
+        ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.derived/pressure/
+          air.mon.mean.nc
+          hgt.mon.mean.nc
+          rhum.mon.mean.nc
+          shum.mon.mean.nc
+          uwnd.mon.mean.nc
+          vwnd.mon.mean.nc
+          omega.mon.mean.nc
+        ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.dailyavgs/pressure/
+          uwnd.????.nc
+          vwnd.????.nc
+      Subdirectory surface/:
+        ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.derived/surface/
+          air.mon.mean.nc
+        ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.derived/surface_gauss/
+          prate.mon.mean.nc
+        ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.dailyavgs/surface_gauss/
+          prate.sft.gauss.????.nc
+        ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.dailyavgs/other_gauss/
+          ulwrf.ntat.gauss.????.nc
+
```

Comment:
Could we maybe replace the "????" by a *

Reply:
In fact, it is there to mean that is the year, and thus only 4 positions should be there. I will change it to YYYY so it is clearer